metricbeat/module/linux/memory/memory_linux_test.go

```diff
@@ -22,10 +22,22 @@ import (
 
 	"github.com/stretchr/testify/assert"
 
+	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/beats/v7/libbeat/metric/system/resolve"
 	mbtest "github.com/elastic/beats/v7/metricbeat/mb/testing"
 	_ "github.com/elastic/beats/v7/metricbeat/module/linux"
 )
 
+func TestPercents(t *testing.T) {
+	res := resolve.NewTestResolver("./_meta/testdata/")
+	data := common.MapStr{}
+	err := FetchLinuxMemStats(data, res)
+	assert.NoError(t, err, "FetchLinuxMemStats")
+
+	assert.Equal(t, float64(1), data["page_stats"].(common.MapStr)["kswapd_efficiency"].(common.MapStr)["pct"].(float64))
+	assert.Equal(t, float64(0.7143), data["page_stats"].(common.MapStr)["direct_efficiency"].(common.MapStr)["pct"].(float64))
```

Comment:
Is this enough test coverage for all the changes?

I'd be worried about accidental typos in the paths, keys, and such. I think they are all fine but it is hard to know if I would have noticed them all.

Reply:
Agreed, I'll add a few more.